vignettes/wrapping-apis.Rmd

```diff
@@ -442,6 +466,8 @@ get_api_key <- function() {
 Then you could modify `nytimes_books()` to use `get_api_key()` as the default value for `api_key`.
 Since the argument is now optional, we can move it to end of the argument list, since it'll only be needed in exceptional circumstances.
 
+JENNY COMMENT HERE
```

Comment:
Maybe also worth pointing out "to the end and *after `...`*". If indeed that is intentional, but I suspect it is.

Reply:
I was just using `...` casually here, but yes, I would recommend putting it after `...` if it exists (i.e. https://design.tidyverse.org/dots-position.html). Do you think I need to explicitly call that out here?

Reply:
Well, I really like this principle, so I'd take the chance to publicize it. But I also don't think matters much here. It's tangential.